evergreen/config.yml

```diff
@@ -1695,7 +1727,9 @@ buildvariants:
     cmake_build_type: RelWithDebInfo
     enable_tsan: On
   tasks:
-  - name: compile_test
+  # FIXME: tsan is not stable on arm64, fails often with internal errors
+  # - name: compile_test
+  - name: compile_test_object_store
```

Comment:
Do you know if the sync-tests pass on this platform?

Reply:
they do, but way too often they fail #7185